contentcuration/contentcuration/view/internal_views.py

```diff
@@ -124,7 +115,6 @@ def api_finish_channel(request):
         except KeyError:
             raise ObjectDoesNotExist("Missing attribute from data: {}".format(data))
 
-@csrf_exempt
```

Comment:
Does it still work after you remove these? It may then require a CSRF token to be included, which isn't ideal for the API. Let's chat briefly about this.